js_modules/dagster-ui/packages/ui-core/src/runs/RunRoot.tsx

```diff
@@ -166,6 +173,30 @@ export const RunRoot = () => {
 // eslint-disable-next-line import/no-default-export
 export default RunRoot;
 
+const RunAssetTags = (props: {run: RunFragment}) => {
+  const {run} = props;
+  const {data, loading} = useQuery<RunAssetsQuery, RunAssetsQueryVariables>(RUN_ASSETS_QUERY, {
+    variables: {runId: run.id},
+    skip: isHiddenAssetGroupJob(run.pipelineName),
+  });
+
+  if (isHiddenAssetGroupJob(run.pipelineName)) {
+    return <AssetKeyTagCollection useTags assetKeys={assetKeysForRun(run)} />;
+  }
```

Comment:
Render this *instead of* `RunAssetTags` in the hidden asset group job case ( (i.e. at line 152), to avoid having to do the `skip` above? (Then you can also just provide the run ID as a prop instead of the whole fragment.)